src/playbackstrategy/msestrategy.js

```diff
@@ -531,6 +532,10 @@ function MSEStrategy(
         role: "alternate",
         accessibility: { schemeIdUri: "urn:tva:metadata:cs:AudioPurposeCS:2007", value: "1" },
       })
+    } else {
+      mediaPlayer.setInitialMediaSettingsFor("audio", {
+        role: "main",
+      })
```

Comment:
What's thoughts on folding it into one line? Like:

```suggestion
 mediaPlayer.setInitialMediaSettingsFor("audio", enableAD ? /* ad block */ : { role: "main" })
```

I'm thinking down the line that can be extended to a function to get the right settings block for the input value etc

Reply:
I think I may be biased here but I think we should do it

Reply:
Haha no problem, will do that.

Reply:
Done in https://github.com/bbc/bigscreen-player/pull/375/commits/0800f76a8a120104690e13377ebd0cf1f1d53536.